android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -270,14 +270,15 @@ public void onMethodCall(final MethodCall call, final Result result) {
 				Object position = args.get(0);
 				long position2;
 				if (position instanceof Integer) {
-					position2 = (Integer)position;
+					position2 = (Integer) position;
 				} else {
-					position2 = (Long)position;
+					position2 = (Long) position;
 				}
 				seek(position2 == -2 ? C.TIME_UNSET : position2, result);
 				break;
 			case "dispose":
 				dispose();
+				onDispose.run();
```

Comment:
This call to `onDispose.run()` is redundant since the `dispose()` line just above already calls this.